Sources/BadgeGenerator/BadgeGenerator.swift

```diff
@@ -110,3 +110,96 @@ extension UIView {
     }
     
 }
+
+#if DEBUG
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+struct BadgeView: UIViewRepresentable {
+    
+    private let view = UIView()
+    
+    init(view: (UIView) -> ()) {
+        view(self.view)
+    }
+    
+    func makeUIView(context: Context) -> UIView {
+        view.layer.cornerRadius = 15
+        return view
+    }
+    
+    func updateUIView(_ view: UIView, context: Context) {}
+}
+
+fileprivate extension UIColor {
+    static let majorelle = UIColor(red: 0.36, green: 0.40, blue: 0.91, alpha: 1.0)
+    static let flamenco = UIColor(red: 0.92, green: 0.51, blue: 0.30, alpha: 1.0)
+    static let wisteria = UIColor(red: 0.81, green: 0.64, blue: 0.83, alpha: 1.0)
+    static let clamShall = UIColor(red: 0.84, green: 0.70, blue: 0.63, alpha: 1.0)
+    static let lilac = UIColor(red: 1.00, green: 0.66, blue: 0.67, alpha: 1.0)
+}
+
+fileprivate extension UIView {
+    func image(_ image: UIImage) {
+        let imageView = UIImageView(image: image)
+        imageView.tintColor = .black
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            imageView.widthAnchor.constraint(equalToConstant: 40),
+            imageView.heightAnchor.constraint(equalToConstant: 40),
+            imageView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            imageView.centerYAnchor.constraint(equalTo: centerYAnchor)
+        ])
```

Comment:
I prefer to name values that are being used more than once for ease of updating
```suggestion
        imageView.translatesAutoresizingMaskIntoConstraints = false
        let padding:CGFloat = 40
        NSLayoutConstraint.activate([
            imageView.widthAnchor.constraint(equalToConstant: padding),
            imageView.heightAnchor.constraint(equalToConstant: padding),
            imageView.centerXAnchor.constraint(equalTo: centerXAnchor),
            imageView.centerYAnchor.constraint(equalTo: centerYAnchor)
        ])
```

Reply:
this can be discarded as its not being used in this version.